object/error.go

```diff
@@ -1,19 +1,23 @@
 package object
 
+// SplitInfoError is a special errors to show the original object was split to parts.
 type SplitInfoError struct {
 	si *SplitInfo
 }
 
 const splitInfoErrorMsg = "object not found, split info has been provided"
 
+// Error implements error.error interface.
```

Comment:
`implements error interface`? The method can be referred to as `error.Error`, but the interface is just `error` and it's defined as a part of language specification.

Reply:
Updated